app/views/orders/index.html.erb

```diff
@@ -0,0 +1,111 @@
+<h1>Order Fulfillment</h1>
+
```

Comment:
Remove extra white space. One line helps readability, two lines looks like you deleted something and forgot to remove the extra line.